clients/include/device_strided_batch_vector.hpp

```diff
@@ -184,7 +200,8 @@ class device_strided_batch_vector : public d_vector<T, PAD, U>
     //!
     hipError_t transfer_from(const host_strided_batch_vector<T>& that)
     {
-        return hipMemcpy((*this)[0], that[0], sizeof(T) * this->nmemb(), hipMemcpyHostToDevice);
+        return hipMemcpy(
+            this->m_data, that.data(), sizeof(T) * this->nmemb(), hipMemcpyHostToDevice);
```

Comment:
For symmetry, I'd use `this->data()` and `that.data()`, or `this->m_data` and `that.m_data`.

Reply:
Good point